django/conf/global_settings.py

```diff
@@ -157,6 +157,7 @@ def gettext_noop(s):
 LANGUAGE_COOKIE_AGE = None
 LANGUAGE_COOKIE_DOMAIN = None
 LANGUAGE_COOKIE_PATH = '/'
+LANGUAGE_COOKIE_SAMESITE = 'lax'
```

Comment:
I'm not convinced this is worth adding for the language cookie as it's not carrying any critical data (notice how `LANGUAGE_COOKIE_SECURE` and `LANGUAGE_COOKIE_HTTPONLY` don't exist either.